### PR TITLE
Speedup enhancement

### PR DESCRIPTION
**Pull request by zheng-ningxin**
In this pr, we fix several small issues and support the densenet and inception_v3 for the speedup module.

### CODE REVIEW

File: src/sdk/pynni/tests/test_model_speedup.py [hunk 1]
```diff
@@ -145,18 +145,23 @@ def test_speedup_bigmodel(self):
         assert model.backbone2.fc1.in_features == int(orig_model.backbone2.fc1.in_features * SPARSITY)
 
     def test_speedup_integration(self):
-        for model_name in ['resnet18', 'squeezenet1_1', 'mobilenet_v2']:
+        for model_name in ['resnet18', 'squeezenet1_1', 'mobilenet_v2', 'densenet121', 'inception_v3']:
```
Comment (QuanluZhang) [inline src/sdk/pynni/tests/test_model_speedup.py:148]: marks, add warning for unknown mask

File: src/sdk/pynni/tests/test_model_speedup.py [hunk 2]
```diff
@@ -145,18 +145,23 @@ def test_speedup_bigmodel(self):
...
             Model = getattr(models, model_name)
-            net = Model(pretrained=True, progress=False).to(device)
+            if model_name == 'inception_v3':
+                # jit.trace cannot capture the aux_logits path when the net.training is False
+                net = Model(pretrained=True, progress=False, aux_logits=False).to(device)
```
[the remark this answers is not in the document]
Comment (zheng-ningxin) [reply src/sdk/pynni/tests/test_model_speedup.py:152]: Sure~

[the remark this answers is not in the document]
Comment (zheng-ningxin) [reply src/sdk/pynni/tests/test_model_speedup.py]: fixed

File: src/sdk/pynni/nni/compression/torch/speedup/compressor.py [hunk 3]
```diff
@@ -141,6 +141,13 @@ def infer_modules_masks(self):
         """
         for module_name, mask in self.masks.items():
             _logger.debug('Start mask inference from %s', module_name)
+            if module_name not in self.torch_graph.name_to_node:
+                # this module is not traced in the torch_graph,
+                # jit.trace only correctly records functions and
+                # modules which are not data dependent (e.g., do
+                # not have conditionals on data in tensors)
+                # so, if a node is not traced, we just skip it.
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/compression/torch/speedup/compressor.py:149]: better to output warning message here for users to be aware of this

File: src/sdk/pynni/nni/compression/torch/speedup/compressor.py [hunk 4]
```diff
@@ -147,6 +147,7 @@ def infer_modules_masks(self):
                 # modules which are not data dependent (e.g., do
                 # not have conditionals on data in tensors)
                 # so, if a node is not traced, we just skip it.
+                _logger.info('Warning: %s not found in the traced graph, just skip it.', module_name)
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/compression/torch/speedup/compressor.py:150]: _logger.warning('%s has mask, but not found in the traced graph, just skip it.', module_name)
